src/pages/about.astro

```diff
@@ -4,300 +4,219 @@ import CtaButtom from "../components/InlineCtaButton.astro"
-
-#photos img {
-    width: 100%;
-    max-width: 300px;
-    z-index: -99;
-
-}
-
-#photos h3 {
-    font-size: 1.8rem;
+.big-text {
+	width: 75%;
+	font-size: 2rem;
+	font-weight: bold;
 }
 
-picture.photo-left {
-    grid-column-start: 2;
-    grid-column-end: 3;
-    grid-row-start: 1;
-    clip-path: polygon(0 7%, 100% 0, 100% 93%, 0 100%);
-}
-.header.photo-left {
-    grid-column-start: 2;
-    grid-column-end: 3;
-    grid-row-start: 2;
-}
-
-p.photo-left {
-    grid-column-start: 2;
-    grid-column-end: 3;
-    grid-row-start: 3;
-}
-picture.photo-center {
-    grid-column-start: 4;
-    grid-column-end: 5;
-    grid-row-start: 1;
-    clip-path: polygon(0 7%, 100% 0, 100% 93%, 0 100%);
+#about-section {
+	width: 60%;
+	display: flex;
+	flex-direction: row;
+	gap: 40px;
 }
 
-.header.photo-center {
-    grid-column-start: 4;
-    grid-column-end: 5;
-    grid-row-start: 2;
+#about-section h3 {
+	font-size: 1.6rem;
 }
 
-p.photo-center {
-    grid-column-start: 4;
-    grid-column-end: 5;
-    grid-row-start: 3;
+#about-section p {
+	line-height: 25px;
 }
 
-picture.photo-right {
-    grid-column-start: 6;
-    grid-column-end: 7;
-    grid-row-start: 1;
-    clip-path: polygon(0 7%, 100% 0, 100% 93%, 0 100%);
-}
-
-.header.photo-right {
-    grid-column-start: 6;
-    grid-column-end: 7;
-    grid-row-start: 2;
-}
-
-p.photo-right {
-    grid-column-start: 6;
-    grid-column-end: 7;
-    grid-row-start: 3;
+h2 {
+	text-align: center;
+	margin-top: 120px;
+	font-weight: 200;
+	font-size: 2rem;
 }
 
-.pronouns {
-    font-weight: normal;
-    font-size: .75em;
-    letter-spacing: 0.03em;
+.team {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	grid-template-rows: auto;
+	gap: 5%;
+	padding: 5%;
 }
 
-.header {
-    display: flex;
-    flex-direction: column;
-}
+.team img {
+	display: block;
+	width: 100%;
+	max-width: 300px;
+	margin: auto;
 
-.header h3,h4{
-    text-align: center;
+	clip-path: polygon(0 7%, 100% 0, 100% 93%, 0 100%);
 }
 
-.header h3 {
-    margin: 30px 0 20px ;
+.team h3 {
+	font-size: 1.8rem;
+	margin: 30px 0 20px ;
+	text-align: center;
 }
 
-.header h4 {
-    margin: 0 0 20px 0 ;
+.team .pronouns {
+	font-weight: normal;
+	font-size: .75em;
+	letter-spacing: 0.03em;
 }
 
-
-@media screen and (max-width: 1620px) {
-    #photos {
-        grid-template-columns: auto 300px 12% 300px 12% 300px auto
-    }
-
+.team h4 {
+	margin-bottom: 20px;
+	text-align: center;
 }
 
-@media screen and (max-width: 1250px) {
-    #photos {
-        grid-template-columns: auto 250px 8% 250px 8% 250px auto
-    }
-
+.team p {
+	line-height: 1.8;
+	font-weight: 100;
 }
 
 @media screen and (max-width: 870px) {
-    #photos {
-        grid-template-columns: auto 200px 3% 200px 3% 200px auto
-    }
+	.big-text {
+		font-size: 1.3rem;
+	}
 
-    .big-text {
-            font-size: 18pt;
-        }
+	#about-section h3 {
+		font-size: 1.1rem;
+	}
 
-    section h3 {
-        font-size: 15pt;
-    }
-}
+	.team {
+		display: block;
```

Comment:
Switches it back to non-grid mode (rather than a single center column). `display: unset` was used in a few other spots, but I've replaced those with `block` too, as I find it more explicit about what it's doing (and the brpwser support is better).